test/classes/initializers/compilerGenerated/generics/grandParentField.chpl

```diff
@@ -0,0 +1,22 @@
+
+//
+// Ensure that param/type variables from grandparent classes (and beyond) can
+// be used by child classes.
+//
+
+class GrandParent {
+  param rank : int;
+  type idxType;
+}
+
+class Parent : GrandParent {
+  var x : int;
+}
+
+class Child : Parent {
+  var indices : index(rank, idxType);
```

Comment:
Is `index` a library function?

Reply:
It's a standard Chapel type, though not commonly used.

Reply:
Huh, weird.  I'd be tempted to make this a type you've defined in the test itself for clarity, but wouldn't be upset if you left it.